jina/types/sets/document.py

```diff
@@ -26,6 +26,7 @@
 __all__ = ['DocumentSet']
 
 
+@deprecated_class(new_classname="DocumentList")
 class DocumentSet(TraversableSequence, MutableSequence):
```

Comment:
I would prefer removing all the inner code of this class and just type it as
```
@deprecated_class(...)
class DocumentSet(DocumentList):
    pass
```
Same holds true for the other deprecated classes.